accumulator/pollardutil_test.go

```diff
@@ -0,0 +1,45 @@
+package accumulator
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestPollardSerializeDeserialize(t *testing.T) {
+	var p, q Pollard
+	// generate slice of leaf
+	leaves := make([]Leaf, 10)
+	for i := 0; i < len(leaves); i++ {
+		leaves[i].Hash[0] = uint8(i + 1)
```

Comment:
One thing to watch out for: there are parts of the code that treat an all 0 hash as invalid / uninitialized / empty.  To avoid hitting those checks, either start the loop at 1, or add something like `leaves[i].Hash[1] = 0xff`

Reply:
I am using i+1 for each so it starts from 1  , so there should be no 0 hash right. I can  still go ahead with starting with 1.

Reply:
Yup I don't see the all 0 issue here